platform/ui/src/components/DateRange/DateRange.css

```diff
@@ -0,0 +1,107 @@
+.DateRangePicker {
+}
+
+/** CONTAINER STYLES **/
+
+.DateRangePickerInput {
+  @apply bg-transparent border-0 flex;
+}
+
+.DateRangePicker_picker {
+  @apply -mt-1;
+}
+
+/** INPUT DIV STYLES **/
+
+.DateInput {
+  background: transparent;
+  @apply flex flex-1 w-auto;
+}
+
+/** INPUT FIELD COMMON STYLES **/
+
+.DateInput_input {
+  background-image: url('data:image/svg+xml;utf8,<svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24"><g fill="%236b6b6b" fill-rule="evenodd"><path d="M20 20h-4v-4h4v4zm-6-10h-4v4h4v-4zm6 0h-4v4h4v-4zm-12 6h-4v4h4v-4zm6 0h-4v4h4v-4zm-6-6h-4v4h4v-4zm16-8v22h-24v-22h3v1c0 1.103.897 2 2 2s2-.897 2-2v-1h10v1c0 1.103.897 2 2 2s2-.897 2-2v-1h3zm-2 6h-20v14h20v-14zm-2-7c0-.552-.447-1-1-1s-1 .448-1 1v2c0 .552.447 1 1 1s1-.448 1-1v-2zm-14 2c0 .552-.447 1-1 1s-1-.448-1-1v-2c0-.552.447-1 1-1s1 .448 1 1v2z"/></g></svg>');
```

Comment:
I ended up using this `data:image` as background-image for the DateInput because I tried to use the relative svg url and it didn't work. I tried to check the `gatsby-plugin-react-svg` configuration but it didn't work as well. To avoid spending much time in an icon that is pretty much static/won't change, I did this way.